app/src/main/java/nl/hogeschoolrotterdam/projectb/MemoryEditActivity.java

```diff
@@ -57,6 +72,61 @@ protected void onCreate(Bundle savedInstanceState) {
+            @Override
+            public void onClick(View v) {
+                 lastClick= (ImageButton) v;
+                new AlertDialog.Builder(MemoryEditActivity.this)
+                        .setTitle("Add Media")
+                        .setMessage("How do you want to add the media?")
+                        .setPositiveButton("Gallery", new DialogInterface.OnClickListener() {
+                            @Override
+                            public void onClick(DialogInterface dialog, int which) {
+                                Intent mediaIntent = new Intent(Intent.ACTION_PICK, MediaStore.Images.Media.EXTERNAL_CONTENT_URI);
+                                File pictureDirectory = Environment.getExternalStoragePublicDirectory(Environment.DIRECTORY_PICTURES);
+                                String pictureDirectoryPath = pictureDirectory.getPath();
+                                Uri data = Uri.parse(pictureDirectoryPath);
+                                mediaIntent.setDataAndType(data,"image/* video/*");
+                                startActivityForResult(mediaIntent,GALLERY_REQUEST);
+                            }
+
+                        })
+                        .setNegativeButton("Camera", new DialogInterface.OnClickListener() {
+                            @Override
+                            public void onClick(DialogInterface dialog, int which) {
+                                new AlertDialog.Builder(MemoryEditActivity.this)
+                                        .setTitle("Camera")
+                                        .setMessage("How would you like to capture the media?")
+                                        .setPositiveButton("Video", new DialogInterface.OnClickListener() {
+                                            @Override
+                                            public void onClick(DialogInterface dialog, int which) {
+                                                Intent intent = new Intent(MediaStore.ACTION_VIDEO_CAPTURE);
+                                                startActivityForResult(intent,VIDEO_CAMERA_REQUEST);
+                                            }
+                                        })
+                                        .setNegativeButton("Image", new DialogInterface.OnClickListener() {
+                                            @Override
+                                            public void onClick(DialogInterface dialog, int which) {
+                                                Intent intent2 = new Intent(MediaStore.ACTION_IMAGE_CAPTURE);
+                                                startActivityForResult(intent2,IMAGE_CAMERA_REQUEST);
+                                            }
+                                        }).show();
+                            }
+                        })
+                        .show();
+            }
+        };
+
+        cameraButton1 = findViewById(R.id.memory_camera_button1);
```

Comment:
please put all these `findViewById` methods below the rest so that everything is easier to find.